.readme-partials.yml

```diff
@@ -80,6 +80,19 @@ body: |-
   If you already have a "raw" Http `request` object you can assign it to `entry.metadata.httpRequest` directly. More information about
   how the `request` is interpreted as raw can be found in the [code](https://github.com/googleapis/nodejs-logging/blob/15849160116a814ab71113138cb211c2e0c2d4b4/src/entry.ts#L224-L238).
 
+  ## Automatic trace/span Id extraction
```

Comment:
Could you capitalize the "T" in "trace", the "S" and "D" in "Span ID" and the "E" in "Extraction"?